app/models/concerns/hyku_addons/mailboxer_message_mailer_behavior.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+module HykuAddons
+  module MailboxerMessageMailerBehavior
+    extend ActiveSupport::Concern
+
+    def new_message_email(message, receiver)
```

Comment:
I know Codeclimate wants us to combine these into one method, but that seems extreme.  If you can get CodeClimate to ignore this I'd be happy with that.

Reply:
Refactored 👍🏻